db/migrate/20210627175652_add_fields_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddFieldsToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :name, :string
```

Comment:
don't need to change it, but just so you know, you can set the column null value while you are adding it, there's no need to do it on a separated migration. 

`add_column :users, :name, :string, null: false`